crates/shared/src/http_solver_api.rs

```diff
@@ -96,10 +94,14 @@ impl HttpSolverApi for DefaultHttpSolverApi {
             url.query_pairs_mut()
                 .append_pair("ucp_policy", "EnforceForOrders");
         }
-        if self.config.has_use_internal_buffers_parameter {
+        if self.config.use_internal_buffers.is_some() {
```

Comment:
ubernit: This can be written slightly more "Rust"-y as:
```rust
if let Some(use_internal_buffers) = self.config.use_internal_buffers {
    url.query_pairs_mut().append_pair(
        "use_internal_buffers_parameter",
        &use_internal_buffers.to_string(),
    );
}
```

Reply:
Cool, thanks for the tip!